src/libraries/Microsoft.Extensions.Options.ConfigurationExtensions/Microsoft.Extensions.Options.ConfigurationExtensions.sln

```diff
@@ -11,6 +11,20 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Option
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Options.ConfigurationExtensions", "src\Microsoft.Extensions.Options.ConfigurationExtensions.csproj", "{D4C213E7-336F-4601-9F92-1D632D082422}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{4CB60D51-6D36-49DB-A3AC-B45664F39EFC}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Options.ConfigurationExtensions.Tests", "tests\Microsoft.Extensions.Options.ConfigurationExtensions.Tests.csproj", "{5F41C8C2-663D-4D57-9211-2F495D7DFACE}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestUtilities", "..\Common\tests\TestUtilities\TestUtilities.csproj", "{3EFE3DA9-F864-40A3-8892-899D731CC027}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "dependencies", "dependencies", "{78CAAC57-641E-4EBC-AD1F-0AA1034C94A2}"
```

Comment:
Nit: just making a quick comparison style-wise with other sln files in the repo, and we don't seem to have "dependencies" like what you are adding here. e.g. [Primitives](https://github.com/dotnet/runtime/blob/master/src/libraries/Microsoft.Extensions.Primitives/Microsoft.Extensions.Primitives.sln#L5-L18).

Thanks for adding the test project!

Reply:
Okay, I removed the `dependencies` solution folder and moved all together into `test`. https://github.com/dotnet/runtime/pull/39825/commits/ddb583b178e2e14c1ac5fd0a451cabce54fe5262